integration-healthcheck/src/main/java/org/symphonyoss/integration/healthcheck/AsyncCompositeHealthIndicator.java

```diff
@@ -48,19 +34,6 @@
 @Component
 public class AsyncCompositeHealthIndicator implements HealthIndicator {
```

Comment:
You might remove this class and use the 'CompositeHealthIndicator' class provided by Spring Actuator. 

Javadoc: https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/actuate/health/CompositeHealthIndicator.html